### ISSUE TITLE
Improve accessibility of main menu

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#1124]
The main menu is a *lot* more accessible than it used to be, but there is still one big aspect that can be improved. 

While you can currently tab through all the elements of the main menu, you cannot use the arrow keys to navigate through the submenu items once a submenu is activated. Also, while you can tab through these items, the focus is not constrained to the submenu items.

- [ ] Pressing the space bar does not currently open the submenu
- [ ] When a submenu is opened, move focus to the first item
- [ ] Navigate submenu items using the arrow keys

See https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html for more details on implementation

**Comment by diminutivesloop** [comment]
Hey I’d be happy to give a try at fixing this.

**Comment by schalkneethling** [comment]
> Hey I’d be happy to give a try at fixing this.

Sounds great! I am in the process of triaging frontend issues and will be splitting this one up into smaller issues. I will let you know as soon as I have done so and then we can coordinate the work. Thank you for volunteering @diminutivesloop

**Comment by Ryuno-Ki** [comment]
@diminutivesloop Any updates? Even if your PR is not ocmplete, you can submit one and mark it as a draft :)

**Comment by diminutivesloop** [comment]
@Ryuno-Ki Hey yeah I haven’t been able to work on this for a bit but I should be able to move forward with it this week.

**Comment by Ryuno-Ki** [comment]
Take your time.
If you discover that you run out of time, please, let us know.

**Comment by diminutivesloop** [comment]
So I implemented these changes in PR #2997 but after reading [this blog post](https://adrianroselli.com/2017/10/dont-use-aria-menu-roles-for-site-nav.html) I’m wondering if we should take a simpler approach that preserves the existing keyboard navigation experience and removes the “menu” role from the main menu treating it as a simple `nav` element with links.

**Comment by schalkneethling** [comment]
> So I implemented these changes in PR #2997 but after reading [this blog post](https://adrianroselli.com/2017/10/dont-use-aria-menu-roles-for-site-nav.html) I’m wondering if we should take a simpler approach that preserves the existing keyboard navigation experience and removes the “menu” role from the main menu treating it as a simple `nav` element with links. Something similar in behavior to the main navigation on https://www.usa.gov. Thoughts?

Thank you for all your work on this. My apologies for the slowness to respond, I have been heads down on some other items.
I will read the post you have shared. I am also going to ask one of our accessibility folks to assist with the review and making a decision around how to best proceed.

I will get back to you as soon as possible.
